.circleci/config.yml

```
@@ -37,16 +37,10 @@ workflows:
           name: constraints-<< matrix.python_version >>-<< matrix.airflow_version >>
           matrix:
             parameters:
-              python_version: ["3.8", "3.9", "3.10"]
-              airflow_version: ["2.4.3", "2.5.3", "2.6.3"]
-          <<: *main_and_release_branches
-      - generate-constraints-python-3-11:
-          name: constraints-<< matrix.python_version >>-<< matrix.airflow_version >>
-          matrix:
-            parameters:
-              python_version: ["3.11"]
-              airflow_version: ["2.6.3", "2.7.0"]
-          <<: *main_and_release_branches
+              python_version: ["3.8", "3.9", "3.10", "3.11"]
+              airflow_version: ["2.6.3", "2.7.3", "2.8.1"]
+          # <<: *main_and_release_branches
+          <<: *all_branches_and_version_tag
```

```suggestion
          <<: *main_and_release_branches
```
Apply this after verifying that constraints gets successfully generated with this PR